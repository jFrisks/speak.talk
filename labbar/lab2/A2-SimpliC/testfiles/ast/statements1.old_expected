Program
  List
    FunctionCallStmt
      Block
        List
          IdDecl
            ID="x"
          IdDecl
            ID="y"
          Assignment
            IDUse
              ID="x"
            Numeral
              NUMERAL="5"
          Assignment
            IDUse
              ID="y"
            IdUse
              ID="x"
