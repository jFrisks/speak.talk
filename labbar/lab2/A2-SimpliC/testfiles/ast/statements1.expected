Program
  List
    FunctionCallStmt
      Block
        List
          IdDecl ID="x"
          IdDecl ID="y"
          Assignment
            IdUse ID="x"
            Numeral NUMERAL="5"
          Assignment
            IdUse ID="y"
            IdUse ID="x"
